x-pack/plugins/transform/public/app/sections/transform_management/components/action_start/start_action_modal.tsx

```diff
@@ -39,6 +42,19 @@ export const StartActionModal: FC<StartAction> = ({ closeModal, items, startAndC
       buttonColor="primary"
     >
       <p>
+        {hasManagedTransforms ? (
```

Comment:
I don't think we need the warning when starting a managed transform.

Reply:
Removed here https://github.com/elastic/kibana/pull/122305/commits/4820a46f6876f21a96ec6802f9cd4930d7034a7a